scripts/aggregate-themes/disclaimer.txt

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
What is this file intended for?

Reply:
This is a disclaimer to the generated aggregation file to help communicate the file is auto generated and shouldn't be check into git. I added it in a separate txt file so it was easier to read and reason about. 

The file that gets outputted will have this prefixed to it.